wizard.py

```diff
@@ -225,3 +225,32 @@ def default_success(self, data):  # pragma: nocover
             'no_of_orders': self.success.no_of_orders,
             'no_of_products': self.success.no_of_products,
         }
+
+
+class ImportOrderStatesStart(ModelView):
+    "Import Order States Start"
+    __name__ = 'sale.channel.import_order_states.start'
+
+
+class ImportOrderStates(Wizard):
+    """
+    Wizard to import order states for channel
+    """
+    __name__ = 'sale.channel.import_order_states'
+
+    start = StateView(
+        'sale.channel.import_order_states.start',
+        'sale_channel.wizard_import_order_states_start_view_form',
+        [
+            Button('Ok', 'end', 'tryton-ok'),
+        ]
+    )
+
+    def default_start(self, fields):
+        Channel = Pool().get('sale.channel')
+
+        channel = Channel(Transaction().context.get('active_id'))
+
+        channel.import_order_states()
```

Comment:
`import_order_states` should be called in wizard's second state. Then user will get a response after clicking import button.